packages/js/src/generators/library/library.ts

```diff
@@ -72,6 +73,10 @@ export async function projectGenerator(
 
   tasks.push(addProjectDependencies(tree, options));
 
+  if (options.rootProject && options.publishable) {
```

Comment:
We shouldn't need to check root project here. If generating a publishable JS lib, we **_always_** want to make sure that verdaccio has been setup, it just always ends up on the root project
```suggestion
  if (options.publishable) {
```